projects/laji/src/app/shared-modules/laji-form/laji-form/laji-form.component.ts

```diff
@@ -24,6 +24,8 @@ import { TranslateService } from '@ngx-translate/core';
 import { PersonApi } from '../../../shared/api/PersonApi';
 import { combineLatest } from 'rxjs';
 import { Profile } from '../../../shared/model/Profile';
+import LajiForm from 'laji-form/lib/index';
+import { Theme as LajiFormTheme } from 'laji-form/lib/themes/theme';
```

Comment:
I don't remember why we import laji-form asynschronously, but I wonder if importing these here without async breaks something that the async laji-form import is fixing?

Reply:
That was added so that the bundle size for vihko/project main pages could be kept as small as possible. You can run `npm run bundle-report` to get the report to see what's in what bundle (you might have to change dist/browser/stats-es2015.json to dist/browser/stats.json that neme seems to be changing quite often 😂),

I ran those in this branch and all looks good 👍